app/views/programmes/patients.html.erb

```diff
@@ -30,7 +32,12 @@
     <%= render AppSearchResultsComponent.new(@pagy) do %>
       <% @patients.each do |patient| %>
         <%= render AppPatientSearchResultCardComponent.new(
-              patient, link_to: patient_path(patient), programme: @programme, show_year_group: true,
+              patient,
+              link_to: patient_path(patient),
+              programme: @programme,
+              consent_status: @form.consent_status,
```

Comment:
I have some doubt about passing in  `consent_status` and `triage_status` here using the `@form` object - the interface is big enough.

I thought about accessing this data using params inside `AppPatientSearchResultCardComponent`, example `params[:search_form][:consent_status]` but since we already process those params via `SearchForm` I thought maybe using `SearchForm` is better.

What do you think?